src/infinite-list/index.tsx

```diff
@@ -70,8 +71,9 @@ const InfiniteList = (props: InfiniteListProps, ref: any) => {
       }
     )
   );
-  const shouldUseAndroidRTLFix = useMemo(() => {
-    return constants.isAndroidRTL && isHorizontal;
+
+  const shouldFixRTL = useMemo(() => {
+    return isHorizontal && (disableRecycling || constants.isAndroidRTL);
```

Comment:
Are you sure you need here the `disableRecycling` - why  `disableRecycling` affect the RTL fix 
Especially since you do disabled the recycling later on here
```
 disableRecycling={shouldFixRTL || disableRecycling}
```

Reply:
Well, now that I fixed the x when passing disableRecycling I really don't need to actually disable the recycling... so I need another prop.
The point was that if disabling it is the fix I'll let the user pass the prop that controls it, and once the user pass it I'll apply it to the recyclerList AND fix the rtl issue that it causes.
Since I don't want to add a prop that will be redundant after the RN73 upgrade, what do you suggest can control this from the users side?